inc/migration.class.php

```diff
@@ -756,4 +812,44 @@ function updateDisplayPrefs($toadd=array(), $todel=array()) {
       }
    }
 
+   /**
+    * Add a migration SQL query
+    *
+    * @param string $type    Either self::PRE_QUERY or self::POST_QUERY
+    * @param string $query   Query to execute
+    * @param string $message Mesage to display on error, defaults to null
+    *
+    * @return Migration
+    */
+   public function addQuery($type, $query, $message = null) {
```

Comment:
one though here:
$type is not checked, maybe we could set this method private and use dedicated methods or check if type use an existent choice.